File: Php/PetShop.php
```php
<?php
/**
 * Kelas PetShop digunakan untuk mengelola data produk pet shop.
 */
class PetShop
{
    // Atribut private untuk menyimpan data produk
    private $id;
    private $nama;
    private $nkategori;
    private $nharga;
    private $foto;

    /**
     * Konstruktor untuk menginisialisasi objek PetShop.
     * 
     * @param int $id ID produk
     * @param string $nama Nama produk
     * @param string $nkategori Kategori produk
     * @param string $nharga Harga produk
     * @param string $foto Foto produk
     */
    public function __construct($id = 0, $nama = '', $nkategori = '', $nharga = '', $foto = '')
    {
        $this->id = $id;
        $this->nama = $nama;
        $this->nkategori = $nkategori;
        $this->nharga = $nharga;
        $this->foto = $foto;
    }

    // Setter untuk mengatur nilai atribut
    public function setId($id)
    {
        $this->id = $id;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function setnKategori($nkategori)
    {
        $this->nkategori = $nkategori;
    }

    public function setNharga($nharga)
    {
        $this->nharga = $nharga;
    }

    public function setFoto($foto)
    {
        $this->foto = $foto;
    }

    // Getter untuk mendapatkan nilai atribut
    public function getId()
    {
        return $this->id;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function getnKategori()
    {
        return $this->nkategori;
    }

    public function getNharga()
    {
        return $this->nharga;
    }

    public function getFoto()
    {
        return $this->foto;
    }

    /**
     * Metode untuk menampilkan data produk.
     */
    public function tampil()
    {
        // Buat objek produk
        $item1 = new PetShop("1", "Wiskas", "Makanan kucing", "55000", "PetShop.jpg");
        $item2 = new PetShop("2", "Bolt", "Makanan kucing", "19400", "PetShop.jpg");
        $item3 = new PetShop("3", "Poocha", "Makanan basah kucing", "8750", "PetShop.jpg");
        $item4 = new PetShop("4", "Pedigree", "Makanan anjing", "100350", "PetShop.jpg");
        $item5 = new PetShop("5", "Miso", "Makanan anjing", "90000", "PetShop.jpg");

        // Simpan objek dalam array
        $items = array($item1, $item2, $item3, $item4, $item5);

        // Tampilkan data
        $this->outputTable($items);

        // Buat objek produk baru
        $item6 = new PetShop("6", "Cesar", "Makanan anjing", "18500", "PetShop.jpg");
        // Tambahkan objek baru ke array
        array_push($items, $item6);
        // Tampilkan data yang telah diperbarui
        $this->outputTable($items);

        // Hapus objek dari array
        array_splice($items, 5, 1);
        // Tampilkan data setelah penghapusan
        $this->outputTable($items);
    }

    /**
     * Metode untuk menampilkan data produk dalam bentuk tabel.
     * 
     * @param array $items Array objek produk
     */
    private function outputTable($items)
    {
        echo "<br> +-----------------------+ <br>";
        echo "| Data Produk PetShop | <br>";
        echo "+-----------------------+ <br>";
        // Buat tabel
        echo "<table>";
        echo "<tr>
                <th>ID</th>
                <th>Nama</th>
                <th>Kategori</th>
                <th>Harga</th>
                <th>Foto</th>
            </tr>";
        // Tampilkan isi tabel
        foreach ($items as $item) {
            echo "<tr>";
            echo "<td>" . $item->getId() . "</td>";
            echo "<td>" . $item->getNama() . "</td>";
            echo "<td>" . $item->getnKategori() . "</td>";
            echo "<td>" . $item->getNharga() . "</td>";
            echo "<td><img src='image/" . $item->getFoto() . "' alt='" . $item->getNama() . "'</td>";
            echo "</tr>";
        }
        echo "</table>";
    }
}
?>

File: Pyhton/PetShop.py
class PetShop:
    # Konstruktor untuk menginisialisasi atribut PetShop
    def __init__(self, n=0, nama="", nkategori="", nharga=""):
        self.id = n  # ID produk PetShop
        self.nama = nama  # Nama produk PetShop
        self.nkategori = nkategori  # Kategori produk PetShop
        self.nharga = nharga  # Harga produk PetShop

    # Setter methods untuk mengubah nilai atribut
    def set_id(self, n):
        self.id = n  # Mengubah ID produk PetShop

    def set_nama(self, nama):
        self.nama = nama  # Mengubah nama produk PetShop

    def set_nkategori(self, nkategori):
        self.nkategori = nkategori  # Mengubah kategori produk PetShop

    def set_nharga(self, nharga):
        self.nharga = nharga  # Mengubah harga produk PetShop

    # Getter methods untuk mengambil nilai atribut
    def get_id(self):
        return self.id  # Mengambil ID produk PetShop

    def get_nama(self):
        return self.nama  # Mengambil nama produk PetShop

    def get_nkategori(self):
        return self.nkategori  # Mengambil kategori produk PetShop

    def get_nharga(self):
        return self.nharga  # Mengambil harga produk PetShop

    # Method untuk menambahkan data baru
    def tambah(self):
        print("= = = = Menambahkan Data Baru = = = =")
        self.id = int(input("Masukkan ID Anggota PetShop: "))
        self.nama = input("Masukkan Nama produk PetShop: ")
        self.nkategori = input("Masukkan kategori PetShop: ")
        self.nharga = input("Masukkan Harga produk: ")
        print("\nData berhasil ditambahkan!")
        print("= = = = = = = = = = = = = = = = = = =\n")
        return self

    # Method untuk menampilkan data
    def tampil(self, data):
        print("= = =  = List Produk PetShop = = = =")
        if len(data) == 0:
            print("Kosong \n")
        else:
            for item in data:
                print("=================================")
                print(f"ID                 : {item.get_id()}")
                print(f"Nama Produk        : {item.get_nama()}")
                print(f"Kategori Produk    : {item.get_nkategori()}")
                print(f"Harga Produk       : {item.get_nharga()}")
                print("=================================\n")

    # Method untuk mengubah data
    def ubah(self, data):
        self.tampil(data)
        cari = int(input("Masukkan ID produk PetShop yang akan diubah: "))
        flag = 0

        for item in data:
            if cari == item.get_id():
                item.set_nama(input("Masukkan Nama Produk PetShop: "))
                item.set_nkategori(input("Masukkan kategori produk: "))
                item.set_nharga(input("Masukkan harga produk: "))
                print("\nData berhasil diubah!")
                flag = 1
                break

        if flag != 1:
            print("\nID tidak ditemukan!")
        print("= = = = = = = = = = = = = = = = = =\n")

    # Method untuk mencari data
    def cari(self, data):
        self.tampil(data)
        cari = int(input("Masukkan ID produk PetShop yang ingin dicari: "))
        flag = 0

        for item in data:
            if cari == item.get_id():
                print(f"Produk ditemukan dengan ID          : {item.get_id()}")
                print(f"Nama Produk                         : {item.get_nama()}")
                print(f"Kategori Produk                     : {item.get_nkategori()}")
                print(f"Harga Produk                        : {item.get_nharga()}")
                print("=================================\n")
                flag = 1
                break

        if flag == 0:
            print("ID tidak ditemukan!")

    # Method untuk menghapus data
    def hapus(self, data):
        self.tampil(data)
        cari = int(input("Masukkan ID produk PetShop yang ingin dihapus: "))
        flag = 0

        for i in range(len(data)):
            if cari == data[i].get_id():
                del data[i]
                print("\nData berhasil dihapus!")
                flag = 1
                break

        if flag != 1:
            print("\nID tidak ditemukan!")
        print("= = = = = = = = = = = = = = = = = = = = = = = =")

# Contoh penggunaan
if __name__ == "__main__":
    petshop_data = []
    while True:
        print("1. Tambah Produk")
        print("2. Tampilkan Produk")
        print("3. Ubah Produk")
        print("4. Cari Produk")
        print("5. Hapus Produk")
        print("6. Keluar")
        choice = int(input("Pilih opsi: "))

        if choice == 1:
            produk = PetShop()
            petshop_data.append(produk.tambah())
        elif choice == 2:
            PetShop().tampil(petshop_data)
        elif choice == 3:
            PetShop().ubah(petshop_data)
        elif choice == 4:
            PetShop().cari(petshop_data)
        elif choice == 5:
            PetShop().hapus(petshop_data)
        elif choice == 6:
            break
        else:
            print("Pilihan tidak valid!")